app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def calculate_team_total_rating(players: list) -> int:
+    counter = 0
+    for player in players:
+        counter += player.get_rating()
+    return counter
+
+
+def elves_concert(elves: list) -> None:
```

Comment:
It’s better to write exactly what you expect in the `list`. Please specify the correct typing